### PR TITLE
added flag to skip `bundle exec` to `cocoapods` action

### PR DESCRIPTION
**Pull request by birmacher**

### CODE REVIEW

**Comment by birmacher** [comment]
It will solve the problem mentioned in PR #483 by @sethhoward

File: lib/fastlane/actions/install_cocoapods.rb [hunk 1]
```diff
@@ -47,6 +47,11 @@ def self.available_options
                                        env_name: "FL_COCOAPODS_ANSI",
                                        description: "Show output with ANSI codes",
                                        default_value: true),
+          FastlaneCore::ConfigItem.new(key: :skip_bundle_exec,
```
Comment (KrauseFx) [inline lib/fastlane/actions/install_cocoapods.rb]: I'd suggest we use `bundle_exec` instead and have it `true` by default to make it easier :+1:

**Comment by birmacher** [comment]
I've added `use_bundle_exec`. It makes more sense to me, but if you prefer I can remove the `use_` part

**Comment by KrauseFx** [comment]
Looks good to me, thanks :+1:

**Pull request merged**